manifests/o/OCaml/opam/2.2.0/OCaml.opam.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: OCaml.opam
+PackageVersion: 2.2.0
+PackageLocale: en-US
+Publisher: OCaml, Inc.
+PublisherUrl: https://www.ocaml.org/
+PublisherSupportUrl: https://github.com/ocaml/opam/issues
+PrivacyUrl: https://ocaml.org/policies/privacy-policy
+PackageName: opam
+PackageUrl: https://opam.ocaml.org/
+License: LGPL-2.1-only WITH OCaml-LGPL-linking-exception
+LicenseUrl: https://github.com/ocaml/opam/blob/master/LICENSE
+Copyright: Copyright 2012-2024 the opam authors
+CopyrightUrl: https://github.com/ocaml/opam/blob/master/README.md#copyright-and-license
+ShortDescription: A package manager for OCaml.
+Description: Opam is a source-based package manager for OCaml. It supports multiple simultaneous compiler installations, flexible package constraints, and a Git-friendly development workflow.
+Tags:
+- foss
+- ocaml
+- open-source
+- package-manager
+- dependency-manager
```

```suggestion
- dependency-manager
Moniker: ocaml
```
Adding `Moniker: opam` _should_ tell the client to install `OCaml.opam` when a user types `winget install opam` provided no other PackageId in the repo shares this moniker